docs/src/app/components/pages/components/TimePicker/ExampleInternational.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import TimePicker from 'material-ui/lib/time-picker/time-picker';
+
+const TimePickerInternational = () => (
+  <div>
+    <TimePicker
+      hintText="Custom Wordings"
+      wordings={{ok: 'OK', cancel: 'Cancel'}}
```

Comment:
Could you not use English for the i18n example :grimacing:. If you have no other idea. You can use french:
`wordings={{ok: 'OK', cancel: 'Annuler'}}`.